featuretest.js

```diff
@@ -429,8 +429,8 @@ function browserFeatureTest(successCallback) {
   storeSupport('Geolocation API', navigator.geolocation);
   storeSupport('Battery Status API', navigator.getBattery);
   storeSupport('WebAssembly', typeof WebAssembly !== 'undefined');
-  storeSupport('WebVR 1.1', typeof VRDisplay !== 'undefined' && typeof VRPose !== 'undefined' && typeof Navigator !== 'undefined' && typeof Navigator.getVRDisplays !== 'undefined'); // https://immersive-web.github.io/webvr/spec/1.1/
-  storeSupport('WebXR', typeof XR !== 'undefined' && typeof XRDevice !== 'undefined'); // https://immersive-web.github.io/webxr/spec/latest/
+  storeSupport('WebVR 1.1', typeof navigator.getVRDisplays !== 'undefined' && typeof VRDisplay !== 'undefined' && typeof VRPose !== 'undefined'); // https://immersive-web.github.io/webvr/spec/1.1/
+  storeSupport('WebXR', typeof navigator.xr !== 'undefined' && typeof XRDevice !== 'undefined'); // https://immersive-web.github.io/webxr/spec/latest/

```

Comment:
Good catch with the variable vs type names. The `navigator` object might not exist if running in a JS shell, so it's good to keep checking the presence of that as well, otherwise `navigator.getVRDisplays` would throw an exception.

Reply:
yeah, I figured. what about the lines about for Geolocation and Battery? don’t those fail too?

should probably just mock `navigator` or fix all the checks involving `navigator`?

Reply:
Oh good point, there have indeed been such lines already. I think mocking `navigator` might not be good since that would alter the outside environment. It's somewhat rare that the code might get run in a node.js shell, but theoretically might be possible, e.g. if someone has the script present in a test runner or similar, and would want to have it gracefully pass still.

Let me merge this still, since looks like this won't cause any new things like you mention, so such shell-proofing for other environments can then be done later.